vault/token_store.go

```diff
@@ -3295,6 +3295,9 @@ func (ts *TokenStore) handleRevokeOrphan(ctx context.Context, req *logical.Reque
 		return logical.ErrorResponse("missing token ID"), logical.ErrInvalidRequest
 	}
 
+	// TODO #21772 makes the sudo check below redundant, by correcting the TokenStore's PathsSpecial.Root to match this endpoint
+	// However, we are leaving it in place for now, and will remove it later in a second PR, only with vault-core team approval
```

Comment:
```suggestion
	// TODO #21772 makes the sudo check below redundant, by correcting the TokenStore's PathsSpecial.Root to match this endpoint
```
I'd leave this out, don't want to call out specific team in code :)

Reply:
Well, OK, if you prefer. I thought it was helpful to explain the motivation for leaving it in, which makes no sense apart from if the need to have someone else sign off is acknowledged.